zds/member/migrations/0009_profile_show_staff_badge.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+from django.contrib.auth.models import User
+from zds.member.models import Profile
+
+
+def forwards_func(apps, schema_editor):
+    # Check for each user if the staff badge should be displayed
+    users = User.objects.all()
+    for user in users:
+        if user.has_perm('forum.change_post'):
+            try:
+                user_profile = user.profile
+                user_profile.show_staff_badge = True
+                user_profile.save()
```

Comment:
Il vaudrait mieux 

1. Récupérer les id des [users qui ont cette permission](https://stackoverflow.com/questions/378303/how-to-get-a-list-of-all-users-with-a-specific-permission-group-in-django) (demande directement une `values_list` qui est `flat`)
2. Faire directement un truc genre `UserProfile.objects.filter(user__in=).update(show_staff_badge=`